src/foundation.cs

```diff
@@ -3173,9 +3172,12 @@ public interface NSExpression : NSSecureCoding, NSCopying {
 		[Static, Export ("expressionForFunction:arguments:")]
 		NSExpression FromFunction (string name, NSExpression[] parameters);
 
+		[Static, Export ("expressionWithFormat:")]
+		NSExpression FromFormat (string expressionFormat);
+
 		[Static, Export ("expressionWithFormat:argumentArray:")]
-		NSExpression FromFormat (string format, NSExpression [] parameters);
-		
+		NSExpression FromFormat (string format, NSObject [] parameters);
```

Comment:
I think the confusion comes from

`@property (nullable, readonly, copy) NSArray<NSExpression *> *arguments;`

but your tests shows it works... what does `arguments` shows on them ? (I'll try it out later)